src/Sylius/Behat/Context/Ui/Admin/ManagingProductOptionsContext.php

```diff
@@ -0,0 +1,335 @@
+        );
+    }
+
+    /**
+     * @Then there should still be only one product option with :element :value
+     */
+    public function thereShouldStillBeOnlyOneProductOptionWith($element, $value)
+    {
+        $this->iBrowseProductOptions();
+
+        Assert::true(
+            $this->indexPage->isSingleResourceOnPage([$element => $value]),
+            sprintf('Product option with %s %s cannot be found.', $element, $value)
+        );
+    }
+
+    /**
+     * @Then I should be notified that :element is required
+     */
+    public function iShouldBeNotifiedThatElementIsRequired($element)
+    {
+        $this->assertFieldValidationMessage($element, sprintf('Please enter option %s.', $element));
+    }
+
+    /**
+     * @Then the product option with :element :value should not be added
+     */
+    public function theProductoptionWithElementValueShouldNotBeAdded($element, $value)
+    {
+        $this->iBrowseProductOptions();
+
+        Assert::false(
+            $this->indexPage->isSingleResourceOnPage([$element => $value]),
+            sprintf('Product option with %s %s was created, but it should not.', $element, $value)
+        );
+    }
+
+    /**
+     * @Then /^(this product option) should still be named "([^"]+)"$/
+     * @Then /^(this product option) name should be "([^"]+)"$/
+     */
+    public function thisProductOptionNameShouldStillBe(OptionInterface $productOption, $productOptionName)
+    {
+        $this->iBrowseProductOptions();
+
+        Assert::true(
+            $this->indexPage->isSingleResourceOnPage([
+                'code' => $productOption->getCode(),
+                'name' => $productOptionName,
+            ]),
+            sprintf('Product option name %s has not been assigned properly.', $productOptionName)
+        );
+    }
+
+    /**
+     * @Then the code field should be disabled
+     */
+    public function theCodeFieldShouldBeDisabled()
+    {
+        Assert::true(
+            $this->updatePage->isCodeDisabled(),
+            'Code field should be disabled but it is not'
+        );
+    }
+
+    /**
+     * @When I do not add an option value
+     */
+    public function iDoNotAddAnOptionValue()
+    {
+        // Intentionally left blank to fulfill context expectation
+    }
+
+    /**
+     * @Then I should be notified that at least two option values are required
+     */
+    public function iShouldBeNotifiedThatAtLeastTwoOptionValuesAreRequired()
+    {
+        Assert::true(
+            $this->createPage->checkValidationMessageForOptionValues('Please add at least 2 option values.'),
+            'I should be notified that product option needs at least two option values.'
+        );
+    }
+
+    /**
+     * @Then I should see :amount product options in the list
+     */
+    public function iShouldSeeProductOptionsInTheList($amount)
+    {
+        $foundRows = $this->indexPage->countItems();
+
+        Assert::eq(
+            ((int) $amount),
```

Comment:
You could use [this transformer](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Behat/Context/Transform/LexicalContext.php#L42) to avoid casting value here.